probers/ldap/probe.go

```diff
@@ -0,0 +1,33 @@
+package ldap
+
+import (
+	"github.com/nmcclain/ldap"
```

Comment:
mmitton/ldap only has DialTLS() and doesn't provide the DialTLSDialer() which adds timeout functionality.